.gitignore

```diff
@@ -4,3 +4,4 @@
 coverage/*
 notes.md
 tests/fixtures/cassette*
+composer.lock
```

Comment:
i am more worried about the experience of the users of this library than the experience of contributors. even for contributors, things will work fine without the .lock file unless something is horribly broken, in which case we would be glad to know its broken.

but well, anyways. i am not the maintainer of this library, if you are still convinced its a good idea, i can remove this line. i would however really not test on travis-ci with composer install, but keep doing composer update to not have the CI lie about whether things still work.